tencent-cloud/redis/create.go

```diff
@@ -41,13 +43,22 @@ func NewCredential() (*common.Credential, *profile.ClientProfile) {
 	return credential, cpf
 }
 
-func (tr TcloudRedis) Create() error {
+func (tr TCloudRedis) Create() (exists bool, err error) {
+
+	// Init client
 	credential, cpf := NewCredential()
 	client, err := redis.NewClient(credential, tr.Region, cpf)
 	if err != nil {
-		fmt.Printf("new client err: %s\n", err)
-		return err
+		return false, errors.Wrap(err, "new client error")
+	}
+
+	// Check if instance with the same name already exists
+	exists, err = checkInstanceExists(client, tr.InstanceName)
```

Comment:
so for clarity, this will not work currently because InstanceName doesn't work, but we are setting it up for the future?

i'm good with that approach if so.

Reply:
we need to fork and vendor the tcloud sdk. I'll comment it out for now so it's not confusing